features/cellular/framework/AT/AT_CellularDevice.cpp

```diff
@@ -171,44 +171,64 @@ CellularInformation *AT_CellularDevice::open_information(FileHandle *fh)
 void AT_CellularDevice::close_network()
 {
     if (_network) {
-        release_at_handler(&_network->get_at_handler());
-        delete _network;
-        _network = NULL;
+        _network_ref_count--;
```

Comment:
These changes are causing build to failure (all member variables are undefined)

@ARMmbed/mbed-os-wan

Reply:
This is based on https://github.com/ARMmbed/mbed-os/pull/7969, needs to be removed from this release candidate